backend/song/conftest.py

```diff
@@ -62,8 +78,121 @@ def lyric():
 
 @pytest.fixture
 def lyric_request():
-    return LyricRequestFactory(message='need the lyric for this song')
+    return LyricRequestFactory(message='I need the lyric for this song')
 
 @pytest.fixture
 def translation():
     return TranslationFactory()
+
+@pytest.fixture
+def user():
+    return User.objects.create_user(
+            username="test",
+            email="test@nzembo.msc",
+            name="test user",
+            password="strongP@sswor2"
+        )
+
```

Comment:
Man, I like the way we are using factories in this file...
a quick one why not for user?